blocks/rich-text/index.js

```diff
@@ -418,6 +436,50 @@ export class RichText extends Component {
 		};
 	}
 
+	setInsertPosition() {
+		const container = this.getContainerNode();
+		const containerStyle = window.getComputedStyle( container );
+		const marginLeft = get( containerStyle, 'margin-left', 0 );
```

Comment:
Is this reliable? What if there are multiple containers setting margins? Any way we could avoid this? I merely want to avoid breakage if someone updates the CSS of another component.